esmvalcore/cmor/check.py

```diff
@@ -513,6 +517,36 @@ def _check_coords(self):
                 continue
 
             self._check_coord(coordinate, coord, var_name)
+            coords.append((coordinate, coord))
+
+        self._check_coord_ranges(coords)
```

Comment:
Just out of curiosity: why is it better to perform the check like this instead of just rewriting `_check_coord_points` in the same fashion as `_fix_longitude_0_360`?

Reply:
Because like this it can check all coordinates in parallel.